### PR TITLE
Move method to GroupManager

### PR DESCRIPTION
**Pull request by alenkacz**
Summary:
Before actually working on the TasksController I moved part of the code to GroupManager as it was already suggested in the comment.

JIRA issues: MARATHON-7308

### CODE REVIEW

File: src/test/scala/mesosphere/marathon/api/v2/TasksResourceTest.scala [hunk 1]
```diff
@@ -77,7 +77,7 @@ class TasksResourceTest extends UnitTest with GroupCreation {
       Given("no apps")
       config.zkTimeoutDuration returns 5.seconds
       taskTracker.instancesBySpec returns Future.successful(InstanceTracker.InstancesBySpec.empty)
-      groupManager.rootGroup() returns createRootGroup()
```
Comment (ichernetsky) [inline src/test/scala/mesosphere/marathon/api/v2/TasksResourceTest.scala]: Why is this line deleted? Shouldn't the following one be just added?

Comment (meln1k) [reply src/test/scala/mesosphere/marathon/api/v2/TasksResourceTest.scala]: I have the same question.

Comment (alenkacz) [reply src/test/scala/mesosphere/marathon/api/v2/TasksResourceTest.scala]: indexJson method does not call rootGroup(), just indexTxt does so I think it was a dead line of code even before I started touching this so I just removed it...
